adoc/retail_chap_formulas.adoc

```diff
@@ -219,13 +221,16 @@ Use only alphanumeric characters for the branch identifier.
 [[retail.sect.formulas.saltboot]]
 == Saltboot Formula
 
-The Saltboot formula is used to configure disk images and partitioning.
-
+The Saltboot formula is used to configure disk images and partitioning for selected hardware type.
```

Comment:
There's a missing article here somewhere. Possibly: "...partitioning for **the** selected hardware type."?